src/pages/Dashboard.tsx

```
@@ -604,18 +627,38 @@ const Dashboard: React.FC<DashboardProps> = ({
+                    <Skeleton key={i} className="h-8 w-20" />
+                  ))}
+                </div>
+                <div className="flex-1 flex items-end gap-4 px-4">
+                  {Array.from({ length: 12 }).map((_, i) => (
+                    <div
+                      key={i}
+                      className="flex-1 flex flex-col items-center gap-2"
+                    >
+                      <Skeleton
+                        className="w-full rounded"
+                        style={{ height: `${20 + Math.random() * 60}%` }}
```

ESLint

[error] Error: Cannot call impure function during render

`Math.random` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/pages/Dashboard.tsx:644:50
  642 |                       <Skeleton
  643 |                         className="w-full rounded"
> 644 |                         style={{ height: `${20 + Math.random() * 60}%` }}
      |                                                  ^^^^^^^^^^^^^ Cannot call impure function
  645 |                       />
  646 |                       <Skeleton className="h-3 w-12" />
  647 |                     </div>

(react-hooks/purity)